### ISSUE TITLE
If no program is associated with a file when you double click it, the application crashes.

### ISSUE THREAD

**Issue by Krutonium** [IronManMark20/Explore10#6]
ExploreView.xaml.cs

private void openFile(object sender, RoutedEventArgs e)
        {
            FileItem item = (FileItem)sender;
            Process.Start(item._filepath);  < Here
        }

Exception thrown: 'System.ComponentModel.Win32Exception' in System.dll

**Comment by emmatyping** [comment]
Hm. I should look into this. That is a problem.

**Comment by Krutonium** [comment]
Updated the post, didn't mean to post yet.

**Comment by emmatyping** [comment]
Yeah, I was expecting that starting a process without exception checking is a bad idea
